src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -107,6 +111,9 @@ public static Intake getInstance() {
       m_armPivot.append(new MechanismLigament2d("Arm", Constants.IntakeConstants.kArmLength * 3,
           Units.radiansToDegrees(m_armSim.getAngleRads()), 6, new Color8Bit(Color.kYellow)));
 
+  private final RelativeEncoder m_encoder = m_armMotor.getEncoder();
```

Comment:
The closed loop control operates off the Absolute Encoder, not the Relative Encoder, so these aren't used by the PID controller.